### PR TITLE
Store all possible label names from a series list

### PR DESCRIPTION
**Pull request by 4n4nd**
Addresses #20

### CODE REVIEW

File: pkg/dataframe/series.go [hunk 1]
```diff
@@ -215,10 +215,14 @@ func (a *seriesAggregator) getLabelNames() []string {
 		ls  labels.Labels
 		ret []string
 	)
+	// Create a union set (map) of all the label names.
+	lsMap := make(map[string]string)
```
Comment (bwplotka) [inline pkg/dataframe/series.go:219]: Any unit test to show what exactly this fixes? (:

**Review by bwplotka — APPROVED** [review]
LGTM (:

File: pkg/dataframe/series.go (hunk 1 above)
Comment (4n4nd) [reply pkg/dataframe/series.go:219]: Should I create a test for the `series` package?

Comment (bwplotka) [reply pkg/dataframe/series.go:219]: Would be nice, but it's fine to do in next PR. Test would be really important to ensure this code does what we want ;p

**Pull request merged**